projects/kit/components/progress/pipes/progress-gradient.pipe.ts

```diff
@@ -0,0 +1,33 @@
+import {ElementRef, Inject, Pipe, PipeTransform} from '@angular/core';
+import {USER_AGENT} from '@ng-web-apis/common';
+import {isEdgeOlderThan, isIE} from '@taiga-ui/cdk';
+
+@Pipe({
+    name: 'tuiProgressGradient',
+})
+export class TuiProgressGradientPipe implements PipeTransform {
+    private chromiumEdgeBegins = 79;
```

Comment:
There's no need for it to be property on a pipe, move it to global constants then its name would be minified in prod build.

Reply:
done